index.js

```diff
@@ -154,29 +102,19 @@ const zooAnimals = [
  2. Return the sum of those numbers
  */
 
- function add(a, b){
```

Comment:
Awesome job! 
Small nit: I know you're using a, b naming variables since they made you in the consume function. Since they don't explicitly name these variables definitely try to name them something more specific. Instead of a it can be num1